### DISCUSSION TITLE
Toaster URL in Text and Styling

### DISCUSSION THREAD

**Discussion by matsyui** [fkhadra/react-toastify#950] [Ideas]
It will be a cool feature if we add a feature that will allow us to add URL on the text for adding "Learn more", "Get Help", "Report Issue" something like that and also style them.

![Screenshot_1](https://user-images.githubusercontent.com/62344860/235698585-fed78289-db06-4285-a606-0fd00edcc52e.png)

**Comment by bdresdner** [comment]
Hi @mashwishi Did you ever figure out how to do this?

**Reply by matsyui** [reply]
Nope, i just didn't use this anymore and created my own component

**Reply by makibat** [reply]
This is what worked for me:

```
toast.error(
    <p>
      Error: You don't have enough coins!
      <a className="underline" href="/">
        Learn More.
      </a>
    </p>,
    {
      position: 'bottom-right',
      autoClose: 2000,
      closeOnClick: false,
      pauseOnHover: true,
      draggable: true,
      theme: 'dark'
    }
);
```

![image](https://github.com/fkhadra/react-toastify/assets/28572253/75d353ba-7003-42db-8f8d-7191ee67f1b3)

**Reply by bdresdner** [reply]
Thanks for the reply.  I got it working.

**Reply by matsyui** [reply]
I never knew that it supports element. thank you
